### DISCUSSION TITLE
RTMP Interleaving option

### DISCUSSION THREAD

**Discussion by JamesJunsungKim** [HaishinKit/HaishinKit.swift#1154] [Q&A] [answered]
Thank you for your efforts to maintain the library. It's been a big help! 

We got into a situation where the client had a poor network condition, the streaming server noticed that the video bitrate reduced a bit more than the audio bitrate. We thought Haishinkit might not use an option to interleave RTMP. 

I was wondering if there is an option to send data in an interleaving way or it's already set that way.

Thank you in advance

**Comment by shogo4405** [comment, marked as answer]
HaishinKit has no option. This means RTMPSocket will send interleved always.
